music/src/core/compat/global.ts

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
100% talking out of my butt here, but if you call this file 'index.ts', could you then do `import {fetch} from '../core/compat/'`? if that's true, i think i'd prefer that since it's shorter

Reply:
Sure could - the reason I did `global` was just to highlight the fact that it's importing global/window variables, and also because we may need to do other compatibility-related things in the future... not sure if those ideas resonate with you.

Reply:
sure, it's fine, leave it.